vector/src/main/res/values/config.xml

```diff
@@ -15,6 +15,8 @@
 
     <!-- Note: pusher_http_url should have path '/_matrix/push/v1/notify' -->
     <string name="pusher_http_url" translatable="false">https://matrix.org/_matrix/push/v1/notify</string>
+    <!-- Note: pusher_http_url should have path '/_matrix/push/v1/notify' -->
+    <string name="default_push_gateway_http_url" translatable="false">https://matrix.gateway.unifiedpush.org/_matrix/push/v1/notify</string>
```

Comment:
Update the comment?

Reply:
One remark, but an important one, what happen if this service is down?

Reply:
This push gateway is used to forward notifications to users' UnifiedPush endpoints and is not used by the FCM embedded distributor. So if it is down, users using another UnifiedPush distributor than the embedded one (which will be used by most users) won't receive push notifications. Just like users using FCM embedded distributor won't receive push notif if the matrix sygnal server is down.

This push gateway is the unifiedpush [common-proxies](https://github.com/UnifiedPush/common-proxies) on our server. It can be hosted by element/matrix team. It is also possible to write a sygnal pushkin for that: it should be quite easy to write (with a big warning on SSRF).

Reply:
But if I want to use my server, without participation unifiedpush.org?

Reply:
@manchelsi  You can self host a push proxy like gotify or nfty alongside [common-proxies](https://github.com/UnifiedPush/common-proxies) (only required for gotify), install the apps, point them to your server, and tell element to use the app as a distributor.